tests/MetricsTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use Klaviyo\Metrics as Klaviyo;
+use PHPUnit\Framework\TestCase;
+use GuzzleHttp\Client;
+
+class MetricsTest extends TestCase
+{
+    protected $test_KLAVIYO;
+    protected $test_client;
+
+    public function setUp(): void {
+        parent::setUp(); // TODO: Change the autogenerated stub
+
+        $this->test_KLAVIYO = new Klaviyo(
+            'LzPPzc',
+            'pk_857aa1e36cbca43fb873d7404ee716cba9');
```

Comment:
could we make these a bit more spoofier?

Reply:
The tests/ directory is going to be worked on, so will have test keys then, the current files are just ensuring the SDK works correctly by making calls to our API